core/monitor/price/pricemonitoring.go

```diff
@@ -245,9 +245,12 @@ func (e *Engine) GetCurrentBounds() []*types.PriceMonitoringBounds {
 
 	sort.SliceStable(ret,
 		func(i, j int) bool {
-			return ret[i].Trigger.Horizon <= ret[j].Trigger.Horizon &&
-				ret[i].Trigger.Probability.LessThanOrEqual(ret[j].Trigger.Probability)
+			if ret[i].Trigger.Horizon == ret[j].Trigger.Horizon {
+				return ret[i].Trigger.Probability.LessThanOrEqual(ret[j].Trigger.Probability)
```

Comment:
I don't know if it is valid or not but if you can have two triggers with same horizon and same probability, then you need a tie breaker with the AuctionExtension. As it is now, with the LessThanOrEqual you'll have the same problem if you have 2 with same horizon and probability

Reply:
actually this shouldn't be allowed so it would be sufficient to have `ret[i].Trigger.Probability.LessThan(ret[j].Trigger.Probability)`

but would need to add validation in new market validation, maybe as a separate ticket though.

Reply:
sorted that